api/envoy/config/common/mutation_rules/v3/mutation_rules.proto

```diff
@@ -85,6 +85,10 @@ message HeaderMutationRules {
   // cause the ``rejected_header_mutations`` counter to be incremented.
   // Default is false.
   google.protobuf.BoolValue disallow_is_error = 7;
+
+  // If true, allows the processing response to allow route cache clearing.
+  // If false, route cache clearing is ignored.
+  google.protobuf.BoolValue allow_clearing_route_cache = 8;
```

Comment:
This isn't a header mutation rule. We probably need some other top-level attribute restriction.

Reply:
Done.